### PR TITLE
Use FastSchematicReader by default

### PR DESCRIPTION
**Pull request by SirYwell**
## Description

`HashMultiMap` has no guarantees about the order of the values for a key. In our case, this leads to SPONGE_SCHEMATIC being first when iterating in `ClipboardFormats.findByExtension(...)`. We want FAST to come before, therefore we should use a `MultiMap` where values are in the order they are registered. This matches the `registeredFormats` list used by `findByFile`.

The change to the command is needed to allow actually enforcing a different schematic format.

```[tasklist]
### Submitter Checklist
- [x] Make sure you are opening from a topic branch (**/feature/fix/docs/ branch** (right side)) and not your main branch.
- [x] Ensure that the pull request title represents the desired changelog entry.
- [x] New public fields and methods are annotated with `@since TODO`.
- [x] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/.github/blob/main/CONTRIBUTING.md).
```

### CODE REVIEW

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java [hunk 1]
```diff
@@ -313,9 +313,9 @@ public void load(
             Actor actor, LocalSession session,
             @Arg(desc = "File name.")
                     String filename,
-            @Arg(desc = "Format name.", def = "fast")
-                    String formatName,
             //FAWE start - random rotation
+            @Arg(desc = "Format name.", def = "")
+                    String formatName,
```
Comment (dordsor21) [inline worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java:318]: Why this change?

Comment (SirYwell) [reply worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java:318]: Otherwise we don't know whether the user wrote `//schem load <name>` or `//schem load <name> fast`, but those cases need to be handled separately, otherwise it's impossible to to use e.g. `//schem load <name> slow`.

**Review by NotMyFault — APPROVED** [review]

**Review by dordsor21 — APPROVED** [review]
